boards/xtensa/odroid_go/doc/index.rst

```diff
@@ -0,0 +1,139 @@
+
+- Dual core Xtensa microprocessor (LX6), running at 80 -  240MHz
+- 4 MB of PSRAM
+- 802.11b/g/n/e/i
+- Bluetooth v4.2 BR/EDR and BLE
+- 2.4 inch 320x240 TFT LCD
+- Speaker
+- Micro SD card slot
+- Micro USB port (battery charging and USB_UART data communication
+- Input Buttons (Menu, Volume, Select, Start, A, B, Direction Pad)
+- Expansion port (I2C, GPIO, SPI)
+- Cryptographic hardware acceleration (RNG, ECC, RSA, SHA-2, AES)
+
+.. figure:: img/odroid_go.png
+        :align: center
+        :alt: ODROID-GO
+
+        ODROID-Go Game Kit
+
+External Connector
+------------------
+
++-------+------------------+-------------------------+
+| PIN # | Signal Name      | ESP32-WROVER Functions  |
++=======+==================+=========================+
+| 1     | GND              | GND                     |
++-------+------------------+-------------------------+
+| 2     | VSPI.SCK (IO18)  | GPIO18, VSPICLK         |
++-------+------------------+-------------------------+
+| 3     | IO12             | GPIO12                  |
++-------+------------------+-------------------------+
+| 4     | IO15             | GPIO15, ADC2_CH3        |
++-------+------------------+-------------------------+
+| 5     | IO4              | GPIO4, ADC2_CH0         |
++-------+------------------+-------------------------+
+| 6     | P3V3             | 3.3 V                   |
++-------+------------------+-------------------------+
+| 7     | VSPI.MISO (IO19) | GPIO19, VSPIQ           |
++-------+------------------+-------------------------+
+| 8     | VSPI.MOSI (IO23) | GPIO23, VSPID           |
++-------+------------------+-------------------------+
+| 9     | N.C              | N/A                     |
++-------+------------------+-------------------------+
+| 10    | VBUS             | USB VBUS (5V)           |
++-------+------------------+-------------------------+
+
+Programming
+***********
+
+Two components are required in order to build applications for this board: the
+`toolchain`_ and the `SDK`_.  Both are provided by the SoC manufacturer.
+
+The SDK contains headers and a hardware abstraction layer library (provided only
+as object files) that are required for the port to function.
+
+The toolchain is available for Linux, Windows, and Mac hosts and instructions to
+obtain and set them up are available in the ESP-IDF repository, using the
+toolchain and SDK links above.
+
+Set up build environment
+------------------------
+
+With both the toolchain and SDK installed, the Zephyr build system must be
+instructed to use this particular variant by setting the following shell
+variables:
+
+.. code-block:: console
+
+   export ZEPHYR_TOOLCHAIN_VARIANT="espressif"
+   export ESPRESSIF_TOOLCHAIN_PATH="/path/to/xtensa-esp32-elf/"
+
+Flashing
+--------
+
+The usual ``flash`` target will work with the ``odroid_go`` board configuration.
+Here is an example for the :ref:`hello_world` application using CMake:
+
+.. code-block:: console
```

Comment:
Can we use the ``.. zephyr-app-commands`` directive here (see samples/hello_world/README.rst for an example use).